chsdi/tests/varnish/test_varnish.py

```diff
@@ -0,0 +1,239 @@
+import sys
+import os
+import random
+from unittest import TestCase
+from testconfig import config as tc
+
+
+import requests
+
+
+class TestVarnish(TestCase):
+
+    ''' Testing the Varnish 'security' configuration. As some settings are IP address dependant,
+        we use an external HTTP Proxy to make the queries.
+    '''
+
+    def hash(self, bits=96):
+        assert bits % 8 == 0
+        return os.urandom(bits / 8).encode('hex')
+
+    def timestamp(self):
+        return random.randrange(20140101, 20141001)
+
+    def setUp(self):
+        os.environ["http_proxy"] = tc['vars']['http_proxy']
```

Comment:
Can't we set a default proxy? So we could run it without loading a configuration?

e.g. running `buildout/bin/nosetests chsdi/tests/varnish/test_varnish.py' should also work, no?